app/models/webhooks/slack/new_comment.rb

```diff
@@ -0,0 +1,25 @@
+class Webhooks::Slack::NewComment < Webhooks::Slack::Base
+
+  def text
+    I18n.t :"webhooks.slack.new_comment", author: author.name, name: discussion_link
```

Comment:
I reckon we'd want to find a way to link to the exact comment which the message is referring to; the url would be something like

```
/d/<key>?comment=<commentId>
```
Which should scroll you to that comment automagically.